docs/cpan-starjacking.md

```diff
@@ -0,0 +1,149 @@
+![river](../media/starjacking/river.png)
+
+## Quality indicators in CPAN
+Some other indicators can help in the choice but give more informations about *quality*:
+- Issues
+![Issues](../media/starjacking/issues.png)
+- CPANTesters
+![CPAN Testers](../media/starjacking/testers.png)
+- Bus factor
+![Bus factor](../media/starjacking/bus.png)
+
+Interpretation of indicators "issues" and "bus factor" is debatable.
+
+## Types of attacks
+Link popular repository:
+- "StarJacking" - Attach very popular github repository to module
+
+Craft fake popularity:
+- "PlusPlusJacking" - Manipulate ++ indicator
+- "RiverJacking" - Manipulate river score
+
+## StarJacking ![Stars](../media/starjacking/stars.png)
+StarJacking refers to the trust that VCS stars provides to a module. 
+
+Reusing the popularity of another project, unrelated to the module, 
+and exploiting the non-verification of the link (metadatas) in order make a module falsely popular.
+
+Other ecosystems like PyPI and NPM are also impacted by this problem.
+
+Like other ecosystems, we can attach "metadatas" (repository, website, issue tracker...) to CPAN packages.
+
+It has to be done directly in the release metadatas.
+
+Example of using resources metadata pointing to a popular project, for instance in `META.yml`
+```
+META_MERGE   => {
+    resources => {
+        repository => {
+            type => 'git',
+            url  => 'https://github.com/popular/lot-of-stars.git',
+            web  => 'https://github.com/popular/lot-of-stars',
+        },
+        bugtracker => {
+            web => 'https://github.com/popular/lot-of-stars-and-tickets',
+        },
+    },
+},
+```
+
+Read [CPAN add link to version control system](https://perlmaven.com/how-to-add-link-to-version-control-system-of-a-cpan-distributions) for more about how to set the metadatas.
+
+CPAN metadatas are effectively un-verified.
+
+PyPI now implements a mechanism for verifying metadata with [Project Metadatas Verifications](https://docs.pypi.org/project_metadata/#verified-details):
+
+![PyPI verified](../media/starjacking/pypi-verified.png)
+
+Or it can warn you about metadata information being "unverified":
+
+![PyPI unverified](../media/starjacking/pypi-unverified.png)
+
+Back to (Meta)CPAN, the outcome is that GitHub/Gitlab/... numbers are almost invisible in MetaCPAN. For instance, for a module with more than 200 stars in GitHub:
+
+![GitHub numbers in MetaCPAN](../media/starjacking/metacpan-side.png)
+
+Only issues are visible, but how can issues help in StarJacking?
+
+Still, we can get more details on metadata by hovering **Repository**:
+
+![GitHub Metadatas in CPAN](../media/starjacking/github.png)
+
+As a conclusion, CPAN is not impacted even though CPAN modules can provide inaccurate or incorrect metadata.
+
+It's because GitHub's popularity is not strongly enough propagated to (Meta)CPAN, so lying on metadata (in a particular repository) does not bring any real advantage over other modules.
```

Comment:
Since this is a perception problem, I think it's ok to just say MetaCPAN, not (Meta)CPAN.